pepper-apis/src/test/java/org/broadinstitute/ddp/db/dao/QuestionDaoTest.java

```diff
@@ -729,95 +725,6 @@ private GetGenericQuestionData genericGetTextQuestionSetUp(Handle handle) {
                 activityInstanceDto.getGuid(), activityInstanceDto.getCreatedAtMillis(), question, answers);
     }
 
-    @Test
-    public void testGetQuestionByBlockId_success() {
```

Comment:
These unit tests literally just tests a method that we no longer use. So result of the tests has no use to us. I recommend remove these unit tests.